src/store/index.js

```diff
@@ -174,6 +174,26 @@ export const $store = new Vuex.Store({
                 `summits/${summit_id}/locations/${location}/banners?${query}`
             ))
         },
+        loadTemplate(context, location) {
+            var params = new URLSearchParams(window.location.href.split('?')[1])
```

Comment:
the load template should set the template variable of the store check please how its done with the background here
https://github.com/fntechgit/openstacksignage/blob/39a896a54034106a378f4831e8198de65b011698/src/store/index.js#L195
so u create a setTemplate method that contains all logic u duped here 
https://github.com/fntechgit/openstacksignage/blob/38de21f3979f96f0c03453daaeeacf9ecb6eb717/src/ably.js#L57